sqrl-testing/pom.xml

```diff
@@ -12,7 +12,6 @@
   <modules>
     <module>sqrl-integration-tests</module>
     <module>sqrl-flink-1.16</module>
-    <module>sqrl-flink-1.17</module>
```

Comment:
Why remove flink 1.17? It's nice to be able to test flink 1.17 behavior

Reply:
I added it back.